scripts/Game.gd

```diff
@@ -45,3 +45,9 @@ func count_coins_in_save_slot(id):
 
 func count_coins_in_current_save_slot():
     return count_coins_in_save_slot(current_save_slot_id)
+
+func is_level_completed(id):
+    return id in save_slots[current_save_slot_id].levels
+
+func coins_in_level(id):
```

Comment:
I had it like that and removed it because it's used only in context where this is already checked. Should I redo it now or refactor later when it's used on more places? How should it behave? Return 0 or `null`?

Reply:
Hmmm. If the callers are supposed to always check it (this actually may be the case) than I would add an `assert(id in save_slots[current_save_slot_id].levels)` there. If they are not supposed to check it, a default return value might be a good enough solution. After thinking a bit on this I would go the assert way.

Reply:
Alright, since usage of this function isn't that clear just yet, I'm gonna add that assert for now. We'll see what happens with it later on.